### PR TITLE
Feat/toogle navbar alumni

### PR DESCRIPTION
**Pull request by KodingAffanMaulana**

### CODE REVIEW

File: frontend/src/components/Navbar.jsx [hunk 1]
```diff
@@ -21,12 +22,23 @@ export default function Navbar({ data }) {
...
     // setCollapsed(false); // Tutup menu mobile setelah tautan diklik
   };
 
   useEffect(() => {
+    const setInitialActiveLink = () => {
+      const path = window.location.pathname;
+      if (path === '/') {
+        setActiveLink()
+      }
+      console.log(path);
```
Comment (fajarhikmal214) [inline frontend/src/components/Navbar.jsx:35]: Does this console need to be pushed too?

FYI. If you guys want to ask for a code review, you guys can mention their Github username directly in the PR description.

File: frontend/src/components/Navbar.jsx [hunk 2]
```diff
@@ -52,7 +64,8 @@ export default function Navbar({ data }) {
             rect.top <= windowHeight / 2 &&
             rect.bottom >= windowHeight / 2
           ) {
-            setActiveLink(sections[i]);
+            console.log(section[i]);
```
Comment (fajarhikmal214) [inline frontend/src/components/Navbar.jsx:67]: This one too

**Comment by fajarhikmal214** [comment]
And other tips. You can describe what is the PR means and what the changes are for.

File: frontend/src/components/Navbar.jsx (hunk 1 above)
Comment (YMaximum) [reply frontend/src/components/Navbar.jsx:35]: It doesn't seem to be needed. I'll ask affan. Thank you for the tips kang fajar.

**Review by YMaximum — APPROVED** [review]

**Pull request merged**